pkgs/development/tools/mold/default.nix

```diff
@@ -37,11 +37,10 @@ stdenv.mkDerivation rec {
   };
 
   meta = with lib; {
-    description = "A high performance drop-in replacement for existing unix linkers";
+    description = "A high performance drop-in replacement for existing Unix linkers";
     homepage = "https://github.com/rui314/mold";
-    license = lib.licenses.agpl3Plus;
-    maintainers = with maintainers; [ nitsky ];
-    platforms = platforms.unix;
```

Comment:
We need to keep this line to build on darwin.

Reply:
Isn't `platfomrs.unix` already the default?

Reply:
I thought so.

Reply:
Tested, doesn't seem to need it for me, provided the later `broken` marker is removed.

Reply:
> Isn't `platfomrs.unix` already the default?

https://nixos.org/manual/nixpkgs/unstable/#var-meta-platforms

Reply:
Tested, doesn't seem to need it for me, provided the later `broken` marker is removed.

Reply:
> If no platform is specified, the package does not have prebuilt binaries

Ah yes I remember this now.

Reply:
> If no platform is specified, the package does not have prebuilt binaries

Correct me if I'm wrong, but does that mean "if the platform is not specified, then Hydra won't build the package"? I have a fair share of packages which I maintain that don't have `meta.platforms` set, but I can pull their binary cache without any builds.

Reply:
> I have a fair share of packages which I maintain that don't have `meta.platforms` set, but I can pull their binary cache without any builds.

Unless the function defaults to meta.platforms to something like that is the case for go, rust and python.

Reply:
It appears to me that if there's no `platform` set it just builds x86_64 linux.